tools/topology/sof/pipe-eq-iir-volume-capture-16khz.m4

```diff
@@ -14,7 +14,7 @@ include(`pipeline.m4')
 include(`bytecontrol.m4')
 include(`eq_iir.m4')
 
-define(`CONTROL_NAME', Capture Volume)
+define(`CONTROL_NAME', Capture Volume 1)
```

Comment:
This breaks the ALSA control name conventions. Add a prefix instead. Like '2nd Capture Volume' or so.

Reply:
ok